classes/incident.php

```diff
@@ -149,11 +149,31 @@ public static function add()
         $stmt->bind_param("ii", $service, $status_id);
         $stmt->execute();
         $query = $stmt->get_result();
+
+        $query = $mysqli->query("SELECT * FROM services_subscriber WHERE serviceIDFK=" . $service);
+				while($subscriber = $query->fetch_assoc()){
+          $subscriberQuery = $mysqli->query("SELECT * FROM subscribers WHERE subscriberID=" . $subscriber['subscriberIDFK']);
+          while($subscriberData = $subscriberQuery->fetch_assoc()){
+            $telegramID = $subscriberData['telegramID'];
+            $firstname = $subscriberData['firstname'];
+            
+            $tg_message = urlencode('Hi ' . $firstname . chr(10) . 'There is a status update on a service that you have subscribed. <a href="' . WEB_URL . '">View online</a>');
+            $response = json_decode(file_get_contents("https://api.telegram.org/bot" . TG_BOT_API_TOKEN . "/sendMessage?chat_id=" . $telegramID . "&parse_mode=HTML&text=" . $tg_message), true);
+            if($response['ok'] == true){
+              $tgsent = true;
+            }
+        }
+       
+      }
+      if($tgsent){
```

Comment:
Could you please put the above code into function that would return boolean? I think the add function is already bloated enough :slightly_smiling_face:

Reply:
Sure, I'll have a look tomorrow. I've added this only to not fail the test, so it was quick and dirty^^

Reply:
If you wanted you could show a javascript popup info that it didn't send successfully like `$.growl.error({ message: "<error>" });`

Reply:
ah true, will have a look, thx!

Reply:
I hope I understood what you wish. I removed
```php
 if($response['ok'] == true){
              $tgsent = true;
 }
```
and added it to a separate function in `telegram.php` (not pushed to github yet)

Do you want the decision what to do if fail/success in that function too? It would be nice if you can help me with the `$.growl.error({ message: "<error>" });`JS a bit, since I've never really done something in JS, thank you!